lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -512,6 +512,11 @@ CRI_HMRC_KBV_J6:
       targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
     next:
       targetState: EVALUATE_GPG45_SCORES
+    enhanced-verification:
+      targetState: MITIGATION_02_OPTIONS_WITH_F2F_M2B
```

Comment:
This seems to be specific to M2B - don't think this is quite right here as we're not necessarily in the M2B journey